assets/app/scripts/services/template.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .service("TemplateService", function(){
```

Comment:
I feel like this is too general a name if all it's doing is to hold a template in memory between route changes, but I'm having trouble coming up with a good name. Maybe `ProcessTemplateService`?